batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/controller/UserController.java

```diff
@@ -0,0 +1,159 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.constants.AppConstants;
+import com.lftechnology.batch7crud.constants.AttributeConstants;
+import com.lftechnology.batch7crud.constants.UrlConstants;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.services.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/18/16.
+ */
+
+@WebServlet({ "/users/*" })
+public class UserController extends HttpServlet {
+  private static final Logger LOGGER = Logger.getLogger(UserController.class.getName());
+  UserService userService = new UserService();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make "userService" transient or serializable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1948)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this misleading mutable servlet instance fields or make it "static" and/or "final" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2226)